db/seeds.rb

```diff
@@ -1 +1,6 @@
+Family.delete_all
+Group.delete_all
 
+FactoryBot.create_list(:family, 15)
+
+FactoryBot.create_list(:group, 5)
```

Comment:
`seeds.rb` should be used to seed the database with data for Production.
For dummy data to be seeded in development, for development purposes, create a custom rake task.
For example when we used `rake dev:prime`

Reply:
I will! :)